packages/nvidia_nat_langchain/tests/eval/test_utils.py

```
@@ -0,0 +1,344 @@
+
+    def test_score_field_takes_precedence_over_standard_key(self):
+        """When score_field is set, custom schema handling is always used."""
+        result = {"key": "accuracy", "score": 0.95, "comment": "Good"}
+        output = langsmith_result_to_eval_output_item("id_4", result, score_field="score")
+        assert output.score == 0.95
+        assert output.reasoning["raw_output"] == result
+
+    async def test_adapter_uses_score_field(self):
+        """Adapter passes score_field through to result converter."""
+
+        def custom_schema_evaluator(*, inputs=None, outputs=None, reference_outputs=None, **kwargs):
```

Ruff (0.15.1)

[warning] Unused function argument: `inputs`

(ARG001)